domains/Discussions/Controllers/QuestionsGetAnswersController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Question;
+use  Illuminate\Database\Eloquent\Builder;
+use Illuminate\Contracts\Auth\Factory as Auth;
+use Illuminate\Contracts\Pagination\Paginator;
+
+class QuestionsGetAnswersController extends Controller
```

Comment:
Can we rename this class to follow the same pattern as other answers-related controllers? Something like `AnswersIndexController`.